solr/core/src/java/org/apache/solr/search/DocSetQuery.java

```diff
@@ -26,19 +26,29 @@
 import org.apache.lucene.search.ScoreMode;
 import org.apache.lucene.search.Scorer;
 import org.apache.lucene.search.Weight;
+import org.apache.lucene.util.FixedBitSet;
+import org.apache.solr.util.TestInjection;
 
 import java.io.IOException;
 import java.util.Objects;
 
+/**
+ * A class that accesses Queries based on a DocSet
+ * This class was created as a replacement to DocSet.getTopFilter
+ * which has since been removed with Filter
```

Comment:
I don't think we need this? What value will it provide to somebody new looking at the class?

Reply:
That's true, thanks for the correction. I deleted it